### ISSUE TITLE
Identify application by Name / some clean identifier

### ISSUE THREAD

**Issue by dabeeeenster** [sdegutis/mjolnir#179]
I would like to be able to get a handle on an application by name (or similar) so that I can pre-set some commonly used applications to fixed positions. At the moment I believe you can only reference apps by CFBundleIdentifier which is liable to change?

**Comment by sdegutis** [comment]
@dabeeeenster You can loop through them and look at either their bundleid or their title. That should work, right?

**Comment by sdegutis** [comment]
@dabeeeenster 

``` lua
local app = fnutils.find(application.runningapplications(), function(app) return app:title() == "Google Chrome" end)
```

**Comment by sdegutis** [comment]
Hmm. Sounds like `fnutils.each` is broken.

**Comment by sdegutis** [comment]
Nope, `fnutils.each` is fine. Looks like `application.runningapplications` is broken!!

**Comment by sdegutis** [comment]
Okay this is fixed on master (see #181).

**Comment by sdegutis** [comment]
:+1:
